### DISCUSSION TITLE
Is it possible to add two inStock labels to monitor from the store file?

### DISCUSSION THREAD

**Discussion by lozbinks** [jef/streetmerchant#1864] [Help]
As per the title, I would like to know if it is possible to add two

 inStock labels to the store file for monitoring?

For example the particular store I am monitoring has the label of "In Stock" but also has a "Pre-Order" & 'Back Order' label which I would like to be monitored if possible.

Here is the label in question on the stores website.

![back order label](https://user-images.githubusercontent.com/78455265/106683333-386f3000-65bc-11eb-9025-19ff45eb1ed7.JPG)

This is what I have tried in the store file but unfortunately without success.

![store labels](https://user-images.githubusercontent.com/78455265/106683564-a3b90200-65bc-11eb-8999-fea94975b8dc.JPG)

Any help would be greatly appreciated.

Also thank you very much to @Jef for creating such an awesome program!

**Comment by jef** [comment]
Yes, that should work just fine. Do you mind sending me the page you're after and I can help with the selector to see if it's correct!

**Reply by lozbinks** [reply]
Thank you for the reply Jef, certainly it's on ShopTo's website

https://www.shopto.net/en/xbshw01-xbox-series-s-console-p510759/

They also put these labels up for pre orders on their Xbox Series X and PS5 page's from time to time as well, which can be found here

https://www.shopto.net/en/xbxhw01-xbox-series-x-p191471/

https://www.shopto.net/en/ps5hw01-playstation-5-console-p191472/

**Reply by jef** [reply]
It looks like `.orderbox_inventory` is all of this text:

![image](https://user-images.githubusercontent.com/12074633/107570693-e465dc00-6bb7-11eb-967d-78aec8d41f2e.png)

If it finds any of the words you put into `text`, then it will report as in stock.

**Reply by lozbinks** [reply]
Thanks Jef! 

Apologies for the late reply, only just spotted this.

That is what I had tried in the stores ts file but it didn't seem to pick it up for some reason, have I done something wrong by any chance?

![shopTo ts file](https://user-images.githubusercontent.com/78455265/108535299-13c0cb00-72d3-11eb-85e5-adf964802b89.JPG)
